apps/system/js/card.js

```diff
@@ -130,9 +132,11 @@
    */
   Card.prototype._populateViewData = function() {
     var app = this.app;
-    this.title = (app.isBrowser() && app.title) ? app.title : app.name;
-    this.sslState = app.getSSLState();
+    var hostname = new URL(app.url || app.origin).hostname;
```

Comment:
I think we should use app.config.url here instead of app.origin because app.origin doesn't actually work - it doesn't update when you navigate to a different origin.